worker.sh

```diff
@@ -0,0 +1,2 @@
+#! /usr/bin/bash
+docker run --shm-size="1024m" --gpus all --rm -it     -v OUTPUT_DIR:/home/output_dir lightly/worker:latest token=your_awesome_token worker.worker_id=your_worker_id 
```

Comment:
Let's stick to the convention in the rest of the examples (e.g. https://docs.lightly.ai/docs/install-lightly#register-the-lightly-worker)
- no output volume mounting
- pass token using environment variable
- use `{}` and capital letters for the placeholders
- use linebreak

All together this would look like this:
```bash
docker run --shm-size="1024m" --gpus all --rm -it -e LIGHTLY_TOKEN={MY_LIGHTLY_TOKEN} \ 
    lightly/worker:latest token=your_awesome_token worker.worker_id={MY_WORKER_ID}
```